e2e/support/bamboo/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+# This docker compose file is used to create a dockerized environment when running E2E tests on Bamboo.
```

Comment:
We don’t need the `/e2e/support/bamboo` directory unless you’re running tests on Bamboo.